### PR TITLE
Teardown bug

### PR DESCRIPTION
**Pull request by hugoesthere**
Incorrect message being printed

@kevgo @trushton

### CODE REVIEW

File: exo-deploy/src/deployers/aws-deployer.ls [hunk 1]
```diff
@@ -54,7 +54,7 @@ class AwsDeployer
 
   teardown: ({nuke}, done) ->
     @terraform-file-builder.generate-provider-credentials!
-    process.stdout.write "terraform starting nuke from AWS"
+    process.stdout.write "terraform starting teardown from AWS"
```
Comment (kevgo) [inline exo-deploy/src/deployers/aws-deployer.ls]: Is it a teardown __from__ AWS of a teardown __of__ AWS?

Comment (hugoesthere) [reply exo-deploy/src/deployers/aws-deployer.ls]: Well, it's actually both. If `nuke` is set in the parameters, it's teardown of application infrastructure including the hosted zone. If not, it's teardown from AWS. That distinction is already made in earlier print statements, this statement is just supposed to say "we're starting teardown now".

Comment (kevgo) [reply exo-deploy/src/deployers/aws-deployer.ls]: Maybe say "terraform starting AWS teardown" then? "teardown from" sounds weird, grammar wise

**Comment by hugoesthere** [comment]
@kevgo what do you think?

**Review by kevgo — APPROVED** [review]
Much better now!

**Pull request closed without merging**